extensions/cornerstone/src/hps/mammo.ts

```diff
@@ -0,0 +1,57 @@
+export const mgCC = {
+  id: 'mgCC',
+  name: 'mammo-cc',
+  imageLoadStrategy: 'interleaveTopToBottom',
+  protocolMatchingRules: [
+    { attribute: 'ModalitiesInStudy', constraint: { contains: ['MG', 'DX'] } },
+  ],
+  displaySetSelectors: {
+    isRCC: {
+      seriesMatchingRules: [
+        { weight: 1, attribute: 'isRCC', constraint: { equals: { value: true } } },
+      ],
+    },
+    isLCC: {
+      seriesMatchingRules: [
+        { weight: 1, attribute: 'isLCC', constraint: { equals: { value: true } } },
+      ],
+    },
+  },
+  stages: [
+    {
+      name: 'default',
+      viewportStructure: { layoutType: 'grid', properties: { rows: 1, columns: 2 } },
+      viewports: [{ displaySets: [{ id: 'isRCC' }] }, { displaySets: [{ id: 'isLCC' }] }],
+    },
+  ],
+  numberOfPriorsReferenced: -1,
+};
+
+export const mgMLO = {
```

Comment:
Is there a reason you aren't using the view code matcher?  See mammoDisplaySetSElectors
The code looks like:
const LCCSeriesMatchingRules = [
  {
    weight: 10,
    attribute: 'ViewCode',
    constraint: {
      contains: 'SCT:399162004',
    },
  },
  {
    weight: 5,
    attribute: 'PatientOrientation',
    constraint: {
      contains: 'L',
    },
  },
  {
    weight: 20,
    attribute: 'SeriesDescription',
    constraint: {
      contains: 'L CC',
    },
  },
];